homeassistant/components/notion/__init__.py

```diff
@@ -170,6 +170,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     entry_updates["data"][CONF_REFRESH_TOKEN] = client.refresh_token
     entry_updates["data"][CONF_USER_UUID] = client.user_uuid
 
+    hass.config_entries.async_update_entry(entry, **entry_updates)
```

Comment:
Shouldn't we still check that they have changed?

Reply:
Yes, apologies—moving too fast.